deps/rabbitmq_cli/lib/rabbitmq/cli/core/command_modules.ex

```diff
@@ -74,8 +74,31 @@ defmodule RabbitMQ.CLI.Core.CommandModules do
           []
       end
 
+    all_plugins = PluginsHelpers.list(opts)
```

Comment:
This change means we either have to always have the plugins directory set, including in all kinds of test environments, or avoid returning an error when the directory does not exist.